src/main/java/org/aksw/gerbil/dataset/impl/generic/Generic_NamedEntity.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * This file is part of General Entity Annotator Benchmark.
+ *
+ * General Entity Annotator Benchmark is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * General Entity Annotator Benchmark is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with General Entity Annotator Benchmark.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.aksw.gerbil.dataset.impl.generic;
+
+import java.util.Arrays;
+import java.util.HashSet;
+
+import org.aksw.gerbil.transfer.nif.data.NamedEntity;
+
+public class Generic_NamedEntity extends NamedEntity {
```

Comment:
Again a naming focusing on the XML use case might be better. This looks like it is a more generic class than the `NamedEntity` we are using :wink: